test/integration/steps/utilities/utils.py

```diff
@@ -416,12 +416,22 @@ def check_service_history_change_type(service_id: str, change_type: str):
 
 
 def get_service_history_specified_opening_times(service_id: str) -> dict:
-    """ This function grabs the latest cmsopentimespecified object for a service id and returns it """
+    """This function grabs the latest cmsopentimespecified object for a service id and returns it"""
     service_history = get_service_history(service_id)
     specified_open_times = service_history[list(service_history.keys())[0]]["new"]["cmsopentimespecified"]
     return specified_open_times
 
 
+def get_service_history_standard_opening_times(service_id: str):
+    """This function grabs the latest standard opening times changes from service history"""
+    service_history = get_service_history(service_id)
+    standard_history = []
```

Comment:
What is standard history? maybe rename this variable to standard_opening_times_from_service_history or something else more prescriptive.

Reply:
Done